apps/system/lockscreen/js/lockscreen.js

```diff
@@ -560,6 +568,7 @@
       this._lastUnlockedInterval = now - this._lastUnlockedTimeStamp;
       this._lastLockedTimeStamp = now;
 
+      this.playUnlockSound('/resources/sounds/lock.opus');
```

Comment:
While we check if the sound is enabled to play the unlocking sound, I think for the locking one it needs to follow the same method. Of course this need an UX' opinion, but I think if the spec didn't change a lot it should be done like that.